### PR TITLE
Fix train_subnet networks definition and wait for osnetcfg finished

### PR DESCRIPTION
**Pull request by stuggi**
Sometimes the create of the osnetcfg and related resources take longer and
later tasks fail, e.g. get ControlPlane IP. Lets wait for the osnetcfg to have its resources created correct or fail early.

Also fixes the used networks definition for train_subnet scenario.

### CODE REVIEW

**Review by abays — APPROVED** [review]
/lgtm

File: ansible/local_cert_tasks.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         oc get -n {{ namespace }} osnet ctlplane -o json | jq -r .spec.roleReservations.ControlPlane.reservations[0].ip 
       register: get_control_plane_ip_ctlplane
       until: get_control_plane_ip_ctlplane.stdout|trim|ipaddr != False
-      retries: 60
```
Comment (olliewalsh) [inline ansible/local_cert_tasks.yaml:12]: This normally only takes a couple of retries. Any idea what is causing such a big delay?

Comment (stuggi) [reply ansible/local_cert_tasks.yaml:12]: when I have seen this the config of the nncp took quiet long and the osnetcfg create took longer then usually.

**Review by olliewalsh — APPROVED** [review]
/lgtm

File: ansible/local_cert_tasks.yaml (hunk 1 above)
Comment (stuggi) [reply ansible/local_cert_tasks.yaml:12]: I think I have a better idea. we should probably wait for in the install_networks tasks on  the osnetcfg to have finished before proceed or fail early at that point if there is an issue. that'd make debugging probably easier.

File: ansible/install_networks.yaml [hunk 2]
```diff
@@ -41,14 +41,21 @@
...
     environment: &oc_env
       PATH: "{{ oc_env_path }}"
       KUBECONFIG: "{{ kubeconfig }}"
     # Retry on failure, probably osnet does not exist yet
     retries: 15
     delay: 10
 
+  - name: wait for osnetcfg to have configured interfaces + networks
+    shell: |
+      set -e
```
Comment (olliewalsh) [inline ansible/install_networks.yaml:54]: nit: could just use command: instead of shell + set -e

**Review by olliewalsh — APPROVED** [review]
/lgtm

**Pull request merged**